resources/views/layout/_head.blade.php

```diff
@@ -0,0 +1,10 @@
+<link rel="stylesheet" type="text/css" href="assets/css/bootstrap.min.css">
```

Comment:
em thử tìm hiểu về bower hoặc npm nhé
https://allaravel.com/tutorials/quan-ly-thu-vien-lap-trinh/npm-la-cong-cu-gi/
https://trungquandev.com/toi-da-su-dung-bower-de-quan-ly-cac-thu-vien-code-nhu-the-nao/

Reply:
vâng anh ạ (love2) =))